### ISSUE TITLE
False positive no-member when using Super() with a @property

### ISSUE THREAD

**Issue by bcl** [PyCQA/pylint#783]
The following code:

```python
#!/usr/bin/python3

class Foo(object):
    @property
    def dict(self):
        d = {1: "boo", 2: "hoo"}
        return d

class Bar(Foo):
    @property
    def dict(self):
        d = super(Bar, self).dict
        d.update({1: "moo", 2: "foo"})
        return d
```

returns:

    ************* Module test-dict
    E: 13, 8: Method 'dict' has no 'update' member (no-member)

If I change .dict to .dict() it is happy.

This is with pylint 1.5.2 and astroid 1.4.3

**Comment by PCManticore** [comment]
Thanks! It seems the property access is a bit misunderstood when accessing it from a super call. I'll look into it.

**Comment by bcl** [comment]
Another aspect of this, if you try to check d for membership, eg. if 1 in d: pass you get this error:
    E: 14,16: Value 'd' doesn't support membership test (unsupported-membership-test)

**Comment by PCManticore** [comment]
This should be fixed in astroid by https://github.com/PyCQA/astroid/commit/da683ae017cc5e29a16f63b670272821afdf3b29. I'll release a new version in a couple of hours.
